src/go/pt-galera-log-explainer/types/sst.go

```diff
@@ -0,0 +1,23 @@
+package types
+
+import "time"
+
+type SST struct {
+	Method             string
+	Type               string
+	Joiner             string
+	Donor              string
+	SelectionTimestamp *time.Time
+}
+
+// MustHaveHappenedLocally use the "selected x as donor" timestamp
+// and compare it to the timestamp of donor/joiner wsrep status shift
+// Usually, when it is selected, joiner/donor take a few milliseconds to shift their status
+// This is the most solid way so far to correctly map donor and joiners when concurrents SSTs
+// are running
```

Comment:
😲

Reply:
I'm actually the first one disappointed. There's just not enough metadata printed to make sure.
I actually got cases with 2 SSTs starting at the same time (more or less), each with their donors and joiners, so when files don't have much info yet and we still haven't got each others names (which actually happen on k8s), that's the only reliable way I have left
The false negatives from this method were less impacting for readability, hence why I kept it